### PR TITLE
Ports pandemic to tgui

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann
add: Pandemic is now tgui!
/:cl:
This is just a port to tgui without changing any functionality.
Also contain a lot of lists
Picture of the full thing: 
hi remie

### CODE REVIEW

**Comment by tortellinitony** [comment]
I love the dropper icon

**Comment by lzimann** [comment]
Closest to a syringe that I found, couldn't think of anything else

**Comment by Core0verload** [comment]
"Create virus culture bottle" literally has a bottle on it.

**Comment by Niknakflak** [comment]
Please fix tgui before porting more bullshit to it.
Thanks

**Comment by GunHog** [comment]
Excellent work! From the diff, I can see that it must have been **!!FUN!!** refactoring all that code to be TGUI friendly!

File: code/modules/reagents/chemistry/machinery/pandemic.dm [hunk 1]
```diff
@@ -1,285 +1,180 @@
...
 	icon_state = "mixer0"
 	circuit = /obj/item/weapon/circuitboard/computer/pandemic
-	use_power = 1
+	use_power = TRUE
 	idle_power_usage = 20
 	resistance_flags = ACID_PROOF
 	var/temp_html = ""
-	var/wait = null
+	var/wait
 	var/obj/item/weapon/reagent_containers/beaker = null
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/machinery/pandemic.dm]: Remove this `= null` while you're cleaning up please

Comment (Cyberboss) [inline code/modules/reagents/chemistry/machinery/pandemic.dm]: Also needs to QDEL_NULL in Destroy

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Penguaro** [comment]
Oh @Niknakflak Expecting things to be working fully before modifying them. How positively gauche.

**Pull request merged**